pkg/controllers/dashboard/systemcharts/controller.go

```diff
@@ -102,7 +103,9 @@ func (h *handler) getChartsToInstall() []*chart.Definition {
 				}
 				// add priority class value.
 				if priorityClassName, err := h.chartsConfig.GetPriorityClassName(); err != nil {
-					logrus.Warnf("Failed to get rancher priorityClassName: %v", err)
+					if !apierror.IsNotFound(err) {
+						logrus.Warnf("Failed to get rancher priorityClassName 'rancher-webhook': %v", err)
```

Comment:
```suggestion
						logrus.Warnf("Failed to get rancher priorityClassName for 'rancher-webhook': %v", err)
```

Reply:
☝️?

Reply:
sorry I don't know why my changes weren't in that push